styles/global.css

```diff
@@ -9,46 +8,89 @@ body {
   box-sizing: border-box;
   background-color: whitesmoke;
 }
-
+:root {
+  --main-font: "Hind", san-serif;
+  --paragraph-sans: "Open Sans", sans-serif;
+  --heading-sans: "Hind", san-serif;
+}
 body {
-  font-family: Arial, Helvetica, sans-serif;
   line-height: 1.6;
 }
+
 .headerNav {
   display: flex;
   justify-content: space-between;
   align-items: center;
-  position: absolute;
   width: 100%;
-  /* make sure header always overlay bg image */
-  z-index: 2;
+  background-color: black;
+  position: fixed;
+  height: 60px;
+}
+.logo {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+}
+.logo,
+.navbar,
+.dropbtn,
+.search-box::placeholder {
+  font-family: var(--heading-sans);
+  font-weight: 600;
 }
 
 .logo a {
   text-decoration: none;
-  color: black;
+  color: white;
 }
-.logo h1 {
-  font-size: 24px;
+.logo-h1 {
+  font-size: 16px;
   margin-bottom: 0;
   margin-left: 20px;
+  justify-items: center;
 }
-.logo p {
+.logo-p {
   margin-top: 0;
-  font-size: 14px;
+  font-size: 10px;
+  color: rgb(233, 231, 231);
   text-align: center;
-  margin-left: 20px;
+  margin-left: 16px;
```

Comment:
`.logo h1 margin-left: 20px` 
`.logo 1 margin-left 16px`
this typography placement is clunky. If you ever change the text, the alignment needs to be reworked again. Can you try using flexbox instead

Reply:
yes, I put logo to have display:flex with flex-direction: column and margin 20px.